processor/k8sprocessor/kube/kube.go

```diff
@@ -34,6 +34,8 @@ const (
 	tagNodeName       = "k8s.node.name"
 	tagPodName        = "k8s.pod.name"
 	tagStartTime      = "k8s.pod.startTime"
+	tagHostName       = "k8s.pod.hostname"
+	tagOwnerTemplate  = "k8s.owner.%s"
```

Comment:
We are lack of:
```
  * labels (!)
  * replicaset (!)
  * service name (!)

  * docker container id
  * namespace id
  * container name
  * pod id
```

not sure about terminology, because the fields will be going directly to the receiver with no changes by fluentd